clientapi/clustersmgmt/v1/upgrade_type_type.go

```diff
@@ -23,6 +23,8 @@ package v1 // github.com/openshift-online/ocm-api-model/clientapi/clustersmgmt/v
 type UpgradeType string
 
 const (
+	// Control plane upgrade without validations, relevant only for hosted control plane clusters.
+	UpgradeTypeCVE UpgradeType = "CVE"
```

Comment:
Since we already have control_plane type, and currently this CVE type is only for control_plane, I think we better differentiate so not to create confusion with what is actually upgrading. Suggesting "ControlPlaneCVE".